packages/dx-react-grid/docs/reference/table-header-row.md

```diff
@@ -47,7 +49,30 @@ onWidthChange | (parameters: { shift: number }) => void | An event that initiate
 onWidthDraft | (parameters: { shift: number }) => void | An event that changes the column width used for preview. The initial column width increases by the `shift` value or decreases if `shift` is less than zero.
 onWidthDraftCancel | () => void | An event that cancels changes of column width used for preview.
 draggingEnabled | boolean | Specifies whether drag-and-drop is enabled.
-getMessage | ([messageKey](#localization-messages): string) => string | Returns the text displayed in a sorting control within the header cell.
+getMessage | ([messageKey](#localization-messages): string) => string | Returns the text displayed in a sort label within the header cell.
+children | ReactNode | Header cell children.
+
+### TableHeaderRow.CellContentProps
```

Comment:
The same component in the TableTreeColumn plugin uses the Content name. I am not sure how should we call them, but the name should be consistent.